### ISSUE TITLE
Using reply & sendTextWithMentions ?

### ISSUE THREAD

**Issue by razisek** [open-wa/wa-automate-nodejs#453]
Hi ,
Can we reply message with mention?

Thanks

**Comment by smashah** [comment]
Implemented in 8678da71103638b4e3f2f3cae7db09c2214ec52e

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
@razisek this has been implemented in v 1.9.41:

https://open-wa.github.io/wa-automate-nodejs/classes/client.html#sendreplywithmentions
